crt_portal/cts_forms/mail.py

```diff
@@ -84,15 +84,17 @@ def render_agency_mail(*, complainant_letter: Mail, report, template, extra_ccs=
                 )
 
 
-def render_complainant_mail(*, report, template) -> Mail:
+def render_complainant_mail(*, report, template, action) -> Mail:
     message = template.render_body(report)
 
     if template.is_html:
-        md = markdown.markdown(message, extensions=['extra', 'sane_lists', 'admonition', 'nl2br', CustomHTMLExtension()])
-        html_message = render_to_string('email.html', {'content': md})
+        content = markdown.markdown(message, extensions=['extra', 'sane_lists', 'admonition', 'nl2br', CustomHTMLExtension()])
     else:
-        # replace newlines, \n, with <br> so the API will generate formatted emails
```

Comment:
should we keep this comment?

Reply:
I feel like it's relatively straightforward what we're doing without it. Similar to the 'nl2br' extension on the above line